polar-core/src/formatting.rs

```diff
@@ -568,7 +580,7 @@ pub mod to_polar {
     }
 
     impl Rule {
-        pub fn head_as_string(&self) -> String {
+        pub(crate) fn head_as_string(&self) -> String {
```

Comment:
is there a major significance to this impl being `pub(crate)` where others in the same module are `pub(super)` ?

Reply:
`pub(super)` makes `polar_core::formatting::to_polar::ToPolarString` visible in the `polar_core::formatting` scope so that it's visible here: https://github.com/osohq/oso/blob/cf82d03a1a33afe03aab5278145432b93c962bea/polar-core/src/formatting.rs#L166

The others are `pub(crate)` because they're used elsewhere in `polar_core` outside of `polar_core::formatting`.